app/workers/form5655/vha_submission_job.rb

```diff
@@ -10,8 +10,10 @@ class VHASubmissionJob
 
     def perform(submission_id, user_uuid)
       submission = Form5655Submission.find(submission_id)
-      user = User.find(user_uuid)
+      user = IdentityStore.find(user_uuid)
       DebtManagementCenter::FinancialStatusReportService.new(user).submit_vha_fsr(submission)
+    ensure
```

Comment:
Are you expecting to be able to retry this job? You might want to only destroy the `IdentityStore` model if it succeeds

Reply:
Makes sense